pkg/ingester/ingester.go

```diff
@@ -114,6 +114,9 @@ const (
 	// Value used to track the limit between sequential and concurrent TSDB opernings.
 	// Below this value, TSDBs of different tenants are opened sequentially, otherwise concurrently.
 	maxTSDBOpenWithoutConcurrency = 10
+
+	// This is the closest fitting Prometheus API error code for requests rejected due to limiting.
+	queryLimitingCode = http.StatusServiceUnavailable
```

Comment:
[nit] This is used in a single place. I'm not sure we need a global `const`.

Reply:
Done.